Update README with recommendation to use IPs when DNS is unreliable or slow (#217)
honeycombio/refinery: README.md

## Setting up Refinery

Refinery is designed to sit within your infrastructure where all sources of Honeycomb events (aka spans if you're doing tracing) can reach it.
A standard deployment will have a cluster of two or more Refinery processes accessible via a separate load balancer.

> Refinery processes must be able to communicate with each other to concentrate traces on single servers.

Reviewer: again, using common language, "correct shard for the trace" >> "concentrate traces on single servers"

Author: I'm a bit unclear on the call to action for this comment. Are you suggesting something like the following change?

```suggestion
Refinery processes must be able to communicate with each other to shuttle traces to the correct shard.
```

Should we bring up sharding in this introductory paragraph? Currently, Sharding isn't introduced until the architecture section of this README much farther down.
